clang/test/Driver/sycl-instrumentation.cpp

```diff
@@ -0,0 +1,10 @@
+/// Check that SYCL ITT instrumentation is disabled by default:
+// RUN: %clang -fsycl -### %s 2>&1 \
+// RUN: | FileCheck -check-prefix=CHECK-DEFAULT %s
+// CHECK-DEFAULT-NOT: "-add-instrumentation-calls"
+
+/// Check "fsycl_device_code_add_instrumentation_calls" is passed to sycl post
```

Comment:
AFAIU, sycl post link is no longer touched. Should this test cases be updated? It still mentions -add-instrumentation-calls

Reply:
Indeed, fixed